docs/spec/spec.md

```diff
@@ -197,52 +259,158 @@ channel._
-| --------------- | --------------- | ---------------------------------------------------------------------------------------------------- | ----------------------------- |
-| ref<sup>1</sup> | ObjectReference | The Subscription UID this SubscriberSpec was resolved from.                                          | Must adhere to _Addressable_. |
-| uri<sup>1</sup> | apis.URL        | Either an absolute URL (if ref is not specified). Resolved using the base URI from ref if specified. | Must be a URL.                |
+`KReference` contains enough information to refer to another object.
+
+| Field Name   | Field Type | Requirement | Description                                                                                                              | Default Value                      | Constraints |
+| ------------ | ---------- | ----------- | ------------------------------------------------------------------------------------------------------------------------ | ---------------------------------- | ----------- |
+| `kind`       | `string`   | Required    | [Kind of the referent.](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds) |                                    |             |
+| `namespace`  | `string`   | Optional    | [Namespace of the referent.](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)              | defaulted to the object holding it |             |
+| `name`       | `string`   | Required    | [Name of the referent.](https://kubernetes.io/docs/concepts/oFURLverview/working-with-objects/names/#names)              |                                    |             |
+| `apiVersion` | `string`   | Required    | API version of the referent.                                                                                             |                                    |             |
+
+### duckv1.Destination
+
+`Destination` represents a target of an invocation over HTTP.
+
+| Field Name        | Field Type                                | Description                                                                                          | Constraints                                                |
+| ----------------- | ----------------------------------------- | ---------------------------------------------------------------------------------------------------- | ---------------------------------------------------------- |
+| `ref`<sup>1</sup> | [`duckv1.KReference`](#duckv1.kreference) | Reference to an [`duckv1.Addressable`](#duckv1.addressable).                                         | Must adhere to [`duckv1.Addressable`](duckv1.addressable). |
+| `uri`<sup>1</sup> | [`apis.URL`](#apis.url)                   | Either an absolute URL (if ref is not specified). Resolved using the base URI from ref if specified. | Must be an URL.                                            |
```

Comment:
This 'Either an absolute URL...' reads little funny, seems like it's missing the either / or. Maybe something like:
Either an absolute URL (if ref is not specified) or a relative URL if ref is specified. Resolved...

Reply:
Fixed.